sos/sosreport.py

```diff
@@ -808,9 +809,16 @@ def _exit(self, error=0):
 
     def get_exit_handler(self):
         def exit_handler(signum, frame):
+            self.exit_process = True
             self._exit()
         return exit_handler
 
+    def handle_exception(self, plugname=None, func=None):
+        if self.raise_plugins or self.exit_process:
+            raise
+        if plugname is not None and func is not None:
```

Comment:
@deleisha,  some statements use ```self.handle_exception(plugname, "setup")``` with parameter  and  some statements ```self.handle_exception()``` without parameter, so need be added ```None``` is default for ```plugname``` and  ```func``` to avoid call next statement ```self._log_plugin_exception(plugname, func)```